README.md

```diff
@@ -318,15 +339,26 @@ not synced to the development container. So you have two choices for editing the
 - You can directly edit the temporary copy of the native helper within the development container, for example: `vi /opt/bundler/v1/lib/functions/file_parser.rb`. And then re-run the `cd...` command. This is the fastest way to debug, but any changes won't be saved outside the container.
 - You can edit your local copy, and then [rebuild the native helper](#making-changes-to-native-package-manager-helpers). This will require re-running the dry-run script to pickup the change.
 
+### Debugging Ignore Conditions
+
+Most of the ecosystems in Dependabot-Core support `ignore` conditions which allow a user to specify dependency names or
+versions to exclude from upgrades. The docs for the Dependabot service at GitHub [describe the feature in more detail](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#ignore).
+
+The env var `IGNORE_CONDITIONS` allows passing one or more ignore conditions to the to the [dry-run script](#dry-run-script):
+
+```bash
+IGNORE_CONDITIONS='[{"dependency-name":"*","update-types": ["version-update:semver-major"]}]' \
+bin/dry-run.rb docker test_org/test-dependabot`
+```
 
-### Debugging Security Alerts / Advisories
+### Debugging Security Updates
```

Comment:
I realized "Alerts / Advisories" was a misnomer because we're actually testing the security "update", which is triggered by the alerts, but not the alerts themselves. This is also more consistent with our stated policy that issues about alerts do not belong in this repo.